src/main.ts

```diff
@@ -30,9 +30,10 @@ export async function run(): Promise<void> {
     const subjects = await subjectFromInputs()
 
     // Generate attestations for each subject serially
-    const attestations = await Promise.all(
-      subjects.map(async subject => await attest(subject, visibility))
-    )
+    const attestations: Attestation[] = []
+    for (let subject of subjects) {
+      attestations.push(await attest(subject, visibility))
+    }
```

Comment:
I think the original intent was to process the list serially (see comment) but that is NOT how `.map` works.